### PR TITLE
Dark theme launchprep 1

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Getting down to some nitty gritty now. Here's the Notion page where I'm tracking all the small stuff:
https://www.notion.so/replayio/Dark-Theme-GTM-Plan-dff728b7f58e405a9750fd7fc748488e

Here's a copy/paste from the Notion doc:

**Highest priority**

- [x]  White box in bottom-right corner of sources
- [x]  Sources top right magnifying glass highlights white, should be dark
- [x]  When sources is minimised there’s a white line
- [x]  Elements highlight colours is much too white
- [x]  Headers have big ugly white lines through them
- [x]  Network headers hover state is too dark
- [x]  Network headers has border line on the left
- [x]  Network URL text is too light
- [x]  When breakpoint loads, it has white background ... not looking good

### CODE REVIEW

File: src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
 }
 
 .sources-list {
-  height: 100%;
+  /* height: 100%; */
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:14]: this does not seem dark theme related. Probably should be it's own PR

File: src/ui/components/Timeline/Timeline.css [hunk 2]
```diff
@@ -129,7 +129,6 @@
 .breakpoint-navigation-timeline .progress-line {
   width: 0%;
   height: 5px;
-  border-radius: 4px;
```
Comment (jasonLaster) [inline src/ui/components/Timeline/Timeline.css:132]: this also doesn't seem dark theme related. curious what inspired this

**Review by jasonLaster — APPROVED** [review]
Overall looks good. Two questions

File: src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css (hunk 1 above)
Comment (jonbell-lot23) [reply src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:14]: It is because this line is forcing a visual flaw in dark theme and I couldn't see any regression in light when I removed it

File: src/ui/components/Timeline/Timeline.css (hunk 2 above)
Comment (jonbell-lot23) [reply src/ui/components/Timeline/Timeline.css:132]: Rather than rounding nicely on both the top and bottom, it's only rounding on the top. Probably because the timeline isn't tall enough. I didn't notice it in light theme but it was very noticeable in dark.

Comment (jasonLaster) [reply src/ui/components/Timeline/Timeline.css:132]: Got it - i guess we can fix this up in a follow up. It feels boxy now - but probably better than the alternative

File: src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css (hunk 1 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:14]: Cool - i think @jaril wants to revisit this as well so that the source pane by default is a bit larger

Comment (jonbell-lot23) [reply src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:14]: Ah good to know, thanks. I suspected as much, which is why I commented it out. CC'ing @jaril to make sure I'm not being dumb.

File: src/ui/components/Timeline/Timeline.css (hunk 2 above)
Comment (jonbell-lot23) [reply src/ui/components/Timeline/Timeline.css:132]: Agree it feels boxy, yeah. Looking forward to fixing this soon.

**Comment by jasonLaster** [comment]
closing this since #2 landed which i assume supercedes
